tests/app/main/test_frameworks.py

```diff
@@ -3535,39 +3492,14 @@ def test_should_create_audit_event(self, mandrill_send_email, notify_send_email,
             data={"question": clarification_question, 'framework': 'g-cloud-7'}
         )
 
-    def test_should_create_g7_question_audit_event(self, mandrill_send_email, notify_send_email, s3):
```

Comment:
I think we can combine this with the other 'G7' test by moving the audit event assert up there.